File: mo-system/mo-system-service/src/main/java/ink/ckx/mo/system/mapper/SysUserRoleMapper.java
package ink.ckx.mo.system.mapper;

import com.baomidou.mybatisplus.core.mapper.BaseMapper;
import ink.ckx.mo.system.api.model.entity.SysUserRole;
import org.apache.ibatis.annotations.Mapper;

/**
 * 用户角色持久层
 *
 * @author chenkaixin
 */
@Mapper
public interface SysUserRoleMapper extends BaseMapper<SysUserRole> {

}

File: mo-admin/mo-admin-service/src/main/kotlin/ink/ckx/mo/admin/mapper/SysUserRoleMapper.kt
package ink.ckx.mo.admin.mapper

import com.baomidou.mybatisplus.core.mapper.BaseMapper
import ink.ckx.mo.admin.api.model.entity.SysUserRole
import org.apache.ibatis.annotations.Mapper

/**
 * 用户角色持久层
 *
 * @author chenkaixin
 */
@Mapper
interface SysUserRoleMapper : BaseMapper<SysUserRole>